working/base-interface-final/feature-specification.md

```diff
@@ -406,14 +410,59 @@ This proposal takes the last option where types have exactly the restrictions
 they declare but a lint can be turned on for users who want to be reminded if
 they re-add a capability in a subtype.
 
-Finally, to the actual proposal...
+## Mixin classes
+
+Inline with Dart's permissive default nature, Dart allows any class declaration
+to also be used as a mixin (in spec parlance, it allows a mixin to be "derived
+from a class declaration"), provided the class meets the restrictions that
+mixins require: it must extend `Object` and not define any generative
```

Comment:
"it must extend `Object`" -> "Its immediate superclass must be `Object`"

(The word "extend" is sometimes used transitively, and if taken so every non-`Null` class extends `Object`. The word "superclass" usually isn't transitive. I added "immediate" just to be sure, but I don't think it's technically necessary.)

Also: define -> declare

(Might be O., @eernstg, do we use "define" interchangeably with "declare"? 
I fear that "define" might include implicit members, which would include the default constructor, which is the one constructor a mixin class is allowed to have. With `declare`, we're safer, because you definitely don't declare the default constructor. But we might want to reevaluate phrasing if we introduce default constructor syntax - is it a declaration or not? But that's a future-me's problem.)

Reply:
Done.